src/ui/screens/options.cpp

```diff
@@ -233,7 +233,7 @@ VideoControlScreen::VideoControlScreen(OptionsScreen::State* state)
             ->bind({
                     [] { return sys.prefs->fullscreen(); },
                     [](bool on) {
-                        sys.prefs->set_fullscreen(on);
+                        sys.video->set_fullscreen(on);
```

Comment:
I think there should also be something here in case of cancellation:
https://github.com/arescentral/antares/blob/09387c7023e6eb201c88164cdbc0742f73683862/src/ui/screens/options.cpp#L128-L132

I guess there is also the question of whether fullscreen via the green button/alt-enter should sync into the options UI, and how that should interact with cancellation, but let’s not worry about that for now. The best fix might simply to make this UI canonical and remove green-button/alt-enter.

Reply:
In my PR, those alternate methods do sync with the options UI. Do you still want me to remove green button and alt-Enter?

Reply:
Oh, they do? That’s good then. I guess the first callback is called on each draw operation?

Reply:
It does